Userland/Libraries/LibWeb/CSS/Properties.json

```diff
@@ -126,6 +126,44 @@
       "right",
       "top"
     ],
+    "quirks": [
```

Comment:
Just quickly flicking through this before going to bed, and noticed a typo in the commit message. "not got" instead of "now got". ;^)

Reply:
Thanks! Fixed :^)